declarations/Threads.json

```
@@ -4,20 +4,28 @@
     "Terms of Service": {
       "fetch": "https://help.instagram.com/769983657850450?helpref=faq_content&cms_id=769983657850450",
       "select": [
-        {
-          "startBefore": "[class*=bh]",
-          "endBefore": "[class*=bp]"
-        }
-      ]
+        "div[role=\"main\"]"
+      ],
+      "remove": [
+        "[id*=Related]",
```

We recommend using specific selectors rather than a catch-all for IDs, as it is hard to anticipate what they could match in the future.
```suggestion
        "#Related",
```